app/src/main/java/com/portfolio/tasky/entry/viewModels/RegisterViewModel.kt

```diff
@@ -55,7 +62,9 @@ class RegisterViewModel  @Inject constructor(
         val areFieldsValidated = mutableEmail.value == true && mutablePassword.value == true && mutableName.value == true
         mutableFieldsValid.value = areFieldsValidated  && isNetworkAvailable
     }
-    suspend fun registration(registerModel: RegisterRequest){
-        mutableRegistration.postValue(entryRepository.doRegistration(registerModel))
+    fun registration(registerModel: RegisterRequest){
```

Comment:
Naming suggestion: `register()` since a function does something, so it should be a verb if it doesn't return anything 🤔

Reply:
Okay

Done